### PR TITLE
HN: Add and replace stream

### PR DESCRIPTION
**Pull request by fershady**
add 
 - Alfa & Omega Visión
 - Canal 11
 - Choluteca TV
 - CHTV Canal Hondureño de Televisión
 - Dios Te Ve
 - Dios Te Ve Infantil
 - Maya TV
 - Vida TV

 Replace
 -MVC
-Waldivisión Internacional

### CODE REVIEW

File: channels/hn.m3u [hunk 1]
```diff
@@ -45,10 +57,12 @@ https://emisoras.hn:8081/kerussotv/index.m3u8
 http://lencatelevision.com:8080/hls/ltv1.m3u8
 #EXTINF:-1 tvg-id="LTV.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://i.imgur.com/dprA1P7.png" group-title="",LTV (720p) [Not 24/7]
 https://5e85d90130e77.streamlock.net/6022/6022/playlist.m3u8
-#EXTINF:-1 tvg-id="MetroTV.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="" group-title="",MetroTV (720p)
+#EXTINF:-1 tvg-id="Maya.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://static.wikia.nocookie.net/logopedia/images/9/91/A6287BAC-05B5-4739-9415-01B08CC0DF0E.png" group-title="Local",Maya TV
+https://media.streambrothers.com:1936/8140/8140/playlist.m3u8
```
Comment (freearhey) [inline channels/hn.m3u:61]: image

File: channels/hn.m3u [hunk 2]
```diff
@@ -45,10 +57,12 @@ https://emisoras.hn:8081/kerussotv/index.m3u8
 http://lencatelevision.com:8080/hls/ltv1.m3u8
 #EXTINF:-1 tvg-id="LTV.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://i.imgur.com/dprA1P7.png" group-title="",LTV (720p) [Not 24/7]
 https://5e85d90130e77.streamlock.net/6022/6022/playlist.m3u8
-#EXTINF:-1 tvg-id="MetroTV.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="" group-title="",MetroTV (720p)
+#EXTINF:-1 tvg-id="Maya.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://static.wikia.nocookie.net/logopedia/images/9/91/A6287BAC-05B5-4739-9415-01B08CC0DF0E.png" group-title="Local",Maya TV
```
Comment (freearhey) [inline channels/hn.m3u:60]:
```suggestion
#EXTINF:-1 tvg-id="MayaTV.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://static.wikia.nocookie.net/logopedia/images/9/91/A6287BAC-05B5-4739-9415-01B08CC0DF0E.png" group-title="Local",Maya TV
```

File: channels/hn.m3u [hunk 3]
```diff
@@ -77,5 +91,7 @@ http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/vid
 https://emisoras.hn:8081/tvcopan/index.m3u8
 #EXTINF:-1 tvg-id="UNAHUTV.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://i.imgur.com/De5CSD3.png" group-title="",UNAH UTV (360p) [Not 24/7]
 https://live-utv.unah.edu.hn/web/salida.m3u8
+#EXTINF:-1 tvg-id="VidaTV.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="" group-title="Religious",Vida TV
+http://184.173.181.2:1935/8070/8070/playlist.m3u8
```
Comment (freearhey) [inline channels/hn.m3u:95]: image

File: channels/hn.m3u [hunk 4]
```diff
@@ -33,7 +45,7 @@ http://p1.worldkast.com/ebenezertv2/ngrp:ebenezertv2_all/playlist.m3u8
 https://60417ddeaf0d9.streamlock.net/edntv/videoedntv/playlist.m3u8
 #EXTINF:-1 tvg-id="GloboTV.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://i.imgur.com/UG6Tq7a.png" group-title="",Globo TV (720p) [Not 24/7]
 https://panel.dattalive.com/8122/8122/playlist.m3u8
-#EXTINF:-1 tvg-id="HCH.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://pbs.twimg.com/profile_images/1443944863075028992/AM2NFj1q.jpg" group-title="",HCH Honduras (720p) [Not 24/7]
+#EXTINF:-1 tvg-id="HCH.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://pbs.twimg.com/profile_images/1443944863075028992/AM2NFj1q.jpg" group-title="",HCH Hable Como Habla (720p) [Not 24/7]
```
Comment (freearhey) [inline channels/hn.m3u:48]:
```suggestion
#EXTINF:-1 tvg-id="HCH.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://pbs.twimg.com/profile_images/1443944863075028992/AM2NFj1q.jpg" group-title="",HCH (Hable Como Habla) (720p) [Not 24/7]
```

File: channels/hn.m3u [hunk 5]
```diff
@@ -25,6 +29,14 @@ http://free.fullspeed.tv/iptv-query?streaming-ip=https://dai.ly/x836wjl
 http://190.11.224.235:1935/CANAL36/myStream/playlist.m3u8
 #EXTINF:-1 tvg-id="CholusatSur36.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://i.imgur.com/qRHwhND.png" group-title="",Cholusat Sur 36 (214p) [Not 24/7]
 http://audiotvserver.net:1935/livemedia/cholusat/playlist.m3u8
+#EXTINF:-1 tvg-id="CholutecaTV.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="" group-title="Religious",Choluteca TV
+https://emisoras.hn:8081/cholutecatv/tracks-v1a1/mono.m3u8
+#EXTINF:-1 tvg-id="CHTV.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://chtv.hn/wp-content/uploads/2020/01/512-1.png" group-title="Local",CHTV Canal Hondureño de Televisión
```
Comment (freearhey) [inline channels/hn.m3u:34]:
```suggestion
#EXTINF:-1 tvg-id="CHTV.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://chtv.hn/wp-content/uploads/2020/01/512-1.png" group-title="Local",CHTV (Canal Hondureño de Televisión)
```

**Review by freearhey — CHANGES_REQUESTED** [review]

File: channels/hn.m3u (hunk 1 above)
Comment (LaneSh4d0w) [reply channels/hn.m3u:61]: my

File: channels/hn.m3u (hunk 3 above)
Comment (LaneSh4d0w) [reply channels/hn.m3u:95]: Low bandwidth, but seems to answer.

File: channels/hn.m3u [hunk 6]
```diff
@@ -25,6 +29,14 @@ http://free.fullspeed.tv/iptv-query?streaming-ip=https://dai.ly/x836wjl
 http://190.11.224.235:1935/CANAL36/myStream/playlist.m3u8
 #EXTINF:-1 tvg-id="CholusatSur36.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://i.imgur.com/qRHwhND.png" group-title="",Cholusat Sur 36 (214p) [Not 24/7]
 http://audiotvserver.net:1935/livemedia/cholusat/playlist.m3u8
+#EXTINF:-1 tvg-id="CholutecaTV.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="" group-title="Religious",Choluteca TV
+https://emisoras.hn:8081/cholutecatv/tracks-v1a1/mono.m3u8
```
Comment (LaneSh4d0w) [inline channels/hn.m3u:33]:
```suggestion
https://emisoras.hn:8081/cholutecatv/playlist.m3u8
```

File: channels/hn.m3u [hunk 7]
```diff
@@ -25,6 +29,14 @@ http://free.fullspeed.tv/iptv-query?streaming-ip=https://dai.ly/x836wjl
...
+#EXTINF:-1 tvg-id="CHTV.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://chtv.hn/wp-content/uploads/2020/01/512-1.png" group-title="Local",CHTV (Canal Hondureño de Televisión)
+https://media.streambrothers.com:1936/8158/8158/playlist.m3u8
+#EXTINF:-1 tvg-id="DiosTeVe.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="" group-title="Religious",Dios Te Ve
+https://emisoras.hn:8081/diostevetv/tracks-v1a1/mono.m3u8
```
Comment (LaneSh4d0w) [inline channels/hn.m3u:37]:
```suggestion
https://emisoras.hn:8081/diostevetv/playlist.m3u8
```

File: channels/hn.m3u [hunk 8]
```diff
@@ -25,6 +29,14 @@ http://free.fullspeed.tv/iptv-query?streaming-ip=https://dai.ly/x836wjl
...
+#EXTINF:-1 tvg-id="DiosTeVeInfantil.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="" group-title="Religious",Dios Te Ve Infantil
+https://emisoras.hn:8081/diostevekids/tracks-v1a1/mono.m3u8
```
Comment (LaneSh4d0w) [inline channels/hn.m3u:39]:
```suggestion
https://emisoras.hn:8081/diostevekids/playlist.m3u8
```

File: channels/hn.m3u (hunk 1 above)
Comment (Dum4G) [reply channels/hn.m3u:61]: Not 24/7, couple of hours ago it was offline

File: channels/hn.m3u [hunk 9]
```diff
@@ -45,10 +57,12 @@ https://emisoras.hn:8081/kerussotv/index.m3u8
 http://lencatelevision.com:8080/hls/ltv1.m3u8
 #EXTINF:-1 tvg-id="LTV.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://i.imgur.com/dprA1P7.png" group-title="",LTV (720p) [Not 24/7]
 https://5e85d90130e77.streamlock.net/6022/6022/playlist.m3u8
-#EXTINF:-1 tvg-id="MetroTV.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="" group-title="",MetroTV (720p)
+#EXTINF:-1 tvg-id="MayaTV.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://static.wikia.nocookie.net/logopedia/images/9/91/A6287BAC-05B5-4739-9415-01B08CC0DF0E.png" group-title="Local",Maya TV
```
Comment (Dum4G) [inline channels/hn.m3u:60]:
```suggestion
#EXTINF:-1 tvg-id="MayaTV.hn" tvg-country="HN" tvg-language="Spanish" tvg-logo="https://static.wikia.nocookie.net/logopedia/images/9/91/A6287BAC-05B5-4739-9415-01B08CC0DF0E.png" group-title="Local",Maya TV [Not 24/7]
```

**Review by Dum4G — DISMISSED** [review]

File: channels/hn.m3u (hunk 3 above)
Comment (Dum4G) [reply channels/hn.m3u:95]: image

**Review by MapGuy11 — DISMISSED** [review]

File: channels/hn.m3u (hunk 3 above)
Comment (freearhey) [reply channels/hn.m3u:95]: Still doesn't work for me
image

Comment (LaneSh4d0w) [reply channels/hn.m3u:95]: Most certainly not 24/7

Comment (fershady) [reply channels/hn.m3u:95]: it's better to delete it.

Comment (Dum4G) [reply channels/hn.m3u:95]: Works again. Please mark as Not 24/7 @fershady 
image

**Review by BellezaEmporium — DISMISSED** [review]

**Review by Dum4G — DISMISSED** [review]

**Review by Dum4G — DISMISSED** [review]

**Comment by Dum4G** [comment]
@freearhey

**Review by Dum4G — APPROVED** [review]

**Review by freearhey — APPROVED** [review]

**Pull request merged**